.github/workflows/ubuntu_22_dpcpp.yml

```diff
@@ -0,0 +1,101 @@
+name: Linux (Ubuntu 22.04, Python 3.11, Intel DPC++ Compiler)
+on:
```

Comment:
Should it be run on all PRs? Are there maybe some specific paths like in https://github.com/openvinotoolkit/openvino/blob/master/.github/workflows/dev_cpu_linux_snippets_libxsmm.yml#L5?